austin_parham/python_stack/flask_fundamentals/adv_validation/server.py

```diff
@@ -0,0 +1,23 @@
+# import Flask
+from flask import Flask, render_template, redirect, request, session, flash
+# the "re" module will let us perform some regular expression operations
+import re
+# create a regular expression object that we can use run operations on
+EMAIL_REGEX = re.compile(r'^[a-zA-Z0-9.+_-]+@[a-zA-Z0-9._-]+\.[a-zA-Z]+$')
+app = Flask(__name__)
+app.secret_key = "ThisIsSecret!"
+@app.route('/', methods=['GET'])
```

Comment:
Note: methods is a list which consists of http verbs. By default the method is get. Therefore we don't explicitly specify it every time.